js/exercise-01.js

```diff
@@ -9,3 +9,18 @@
       Log 'name' value.
 
 */
+--1
```

Comment:
Comments in JS are not like this

- [ ] Use correct comment sintax: `// comment` instead of `-- comment`